ntfs_fr.md

```diff
@@ -528,6 +529,48 @@ address_id | chaine | Requis | Identifiant de l'adresse
 street_name | chaine | Requis | Nom de la voierie
 house_number | chaine | Optionnel | Numéro du seuil
 
+### occupancies.txt (optionnel)
+Pour chaque ligne de ce fichier, l'affluence est appliquée à toutes les paires d'horaires successifs d'une circulation remplissant l'ensemble des conditions suivantes :
+
+* les poteaux d'arrêts des deux horaires se trouvent entre `from_stop_area` et `to_stop_area` (inclus)
+* le jour de circulation est inclus entre `from_date` et `to_date`
+* le jour de semaine de la circulation est actif (voir champs `monday`, `tuesday`, etc.)
+* les `departure_time` des deux horaires sont inclus entre `from_time` et `to_time` (inclus)
+* il est possible de monter dans le véhicule sur le premier des deux horaires successifs
+* il est possible de descendre dans le véhicule sur le dernier des deux horaires successifs
+
+Les lignes du fichier sont appliquées dans l'ordre.
+
```

Comment:
Quid du cas : A - B - C - D, affluence fournie seulement pour la section A - D.
Est-ce que l'affluence remonte sur B - C, ou uniquement sur A -B et C - D ?
J'en comprends que les 3 sections auront l'affluence, il y a peut-être moyen d'être encore plus explicite, sinon pas de souci à laisser comme ça.

Reply:
Non, je ne pense pas à un cas oublié, ici.

J'ai du mal à comprendre l'implication/l'usage de vérifier qu'on peut monter/descendre dans la section, je me demande si ça peut pas empêcher de remonter une info utile sur A - B - C "juste" parce que B est sans montée ni descente.

Reply:
Tu mets le doigt sur un cas intéressant... qui est couvert dans ma tête, mais effectivement mal retranscrit dans les spécifications.

- A-C spécifié dans le fichier
- un `trip` qui fait A-B-C mais avec interdiction de montée/descente en B

Dans ma tête, on remplit bien A-B et B-C. Mais avec l'état actuel de la spécification, ce n'est pas le cas. Je pense que mon erreur est de parler de deux horaires successifs... En réalité, c'est toutes successions d'horaire pour lesquels on trouve un début avec montée autorisée et une fin avec descente autorisée et qui est incluse dans les fourchettes d'espace et de temps. Je vais voir pour essayer de corriger.

Reply:
@pbougue @prhod Let me know if it improves or not? (see 4538243).

Reply:
C'est mieux, ça me convient de laisser comme ça, même si j'ai fait une contre-proposition (n'hésite pas à resolve ici si tu veux déplacer la conversation).